Sources/SignalProducer.swift

```diff
@@ -185,6 +205,25 @@ public struct SignalProducer<Value, Error: Swift.Error> {
 	}
 }
 
+extension SignalProducer where Error == AnyError {
+	/// Create a `SignalProducer` that will attempt the given failable operation once for
+	/// each invocation of `start()`.
+	///
+	/// Upon success, the started producer will send the resulting value then
+	/// complete. Upon failure, the started signal will fail with the error that
+	/// occurred.
+	///
+	/// - parameters:
+	///   - operation: A failable closure.
+	public init(_ action: @escaping () throws -> Value) {
+		self.init {
+			return ReactiveSwift.materialize {
```

Comment:
I don't think we actually need this `materialize` alias. I think we can do `Result(attempt: try action())`.